governance-exercises/Interpretation Exercise Response.md

```diff
@@ -0,0 +1,46 @@
+**Do you think there is any ambiguity in these clauses? If so, where, and how would you rewrite the clauses to remove the ambiguity?**
+
+Yes, recent events over the EDSR made it very clear ambiguity exists. Various community members were unsure on if this clause would allow for such a vote. One reasonable interpretation is that this power was meant to be used to fix obvious errors or weird edge cases that break things, but the author clarified that it could be used to deal with broader protocol issues(such as low DSR utilization and declining DAI supply). 
+
+Given the author's intent, I would suggest  the following edits be included in a future Scopes update to clarify that the power is intended to be this broad. To be clear, this does not necessarily mean I believe Governance Facilitators ought to have this power, but that this is the intent of of the author and majority of voting MKR.
+
+**12.1: Bootstrapping Governance Votes**
+
+Governance Facilitators are empowered to use broad judgment when exercising their ability to make direct Atlas interpretations and edits to the Atlas and the Scope Artifacts, when this relates to unintended consequences or mistakes in the affected documents, *as well as when this relates to time-sensitive issues with the Launch Project.*
+
+12.1.1: In case of unintended consequences or mistakes in the Atlas and Scope Artifacts causing Maker Governance to function incorrectly, the Governance Facilitators can at any time run a Governance Poll to enable MKR holders to make a decision that will alleviate and resolve unintended consequences or mistakes.
+
+12.1.2: The Governance Facilitators can at any time propose to edit any content of a Scope Artifact through a Governance Poll.
+
+https://mips.makerdao.com/mips/details/MIP113#12-1-bootstrapping-governance-votes
+
+
+**Situation A**
+
+**Aligned Delegates are not being compensated due to a clear typo elsewhere in the Governance Scope.**
+
+Fixing the typo is within the power of 12.1. This is clearly a "mistake". if participants who are supposed to be paid cannot be paid. I would create polls(1 or 2, depending on the specifics) that fix the typo and backpay the delegates. I would notify Governance through the forums and Discord.
+
+**Situation B**
+
+**A large token holder wants to add logic to the Stability Scope scope more quickly than is possible using the monthly governance cycle. The author claims this change is required to prevent the DAO losing significant value.**
+
+While this is not clearly included in 12.1, there is precedent that the vote should proceed in this case. I would consider the proposal carefully, along with the governance ramifications of posting the poll or rejecting the token holder's request. As noted above, I would also advise changing the wording to clarify this.
```

Comment:
I get what you're saying, and agree that this is a sensible approach. However, the point of these exercises is that you consider the hypothetical situations, and determine a response. The response being: 'I'll think about it carefully and judge it based on the situation' is a good answer, but not a very helpful answer when it comes to evaluating your judgement.

Reply:
For more specific feedback in this case:
* I would maybe have talked a bit about why the precedent complicates the situation, the pros and cons of following the precedent versus a stricter interpretation of the scope.
* Maybe talk about the potential governance ramifications of allowing or rejecting the request?
* Maybe talk about how the changes in wording can improve this situation. (Note, the hypothetical doesn't specify its in any way related to the launch project.)